src/civis/response.py

```diff
@@ -262,7 +253,7 @@ def __eq__(self, other):
         elif isinstance(other, Response):
             return self._data_snake == other._data_snake
         else:
-            raise TypeError(f"Response and {type(other)} can't be compared")
+            return False
```

Comment:
This is borderline between a behavior change and a bug. In a way, I'm loosening the strictness of the equality check here. I ran into multiple cases where a `Response` object was compared to a non-`Response` object, and making it return `False` here instead of raising an error would have made life a lot easier.